### PR TITLE
fixed powershellversion manifest implementation. 

### PR DESCRIPTION
**Pull request by bgelens**
Failed due to faulty rename operation with last commit.

### CODE REVIEW

File: src/InvokePlaster.ps1 [hunk 1]
```diff
@@ -830,7 +830,7 @@ function Invoke-Plaster {
                 if (![string]::IsNullOrWhiteSpace($description)) {
                     $newModuleManifestParams['Description'] = $description
                 }
-                if (![string]::IsNullOrWhiteSpace($psVersion)) {
+                if (![string]::IsNullOrWhiteSpace($PowerShellVersion)) {
```
Comment (rkeithhill) [inline src/InvokePlaster.ps1]: For consistency, can you change the casing to camelCase (powerShellVersion) here and on line 770 where it is assigned?

Comment (bgelens) [inline src/InvokePlaster.ps1]: done.. Sorry!! I'm a real screw up with this kind of casing ;) I'm used to CamelCase as you can see

**Comment by rkeithhill** [comment]
Thanks!

File: src/InvokePlaster.ps1 (hunk 1 above)
Comment (rkeithhill) [inline src/InvokePlaster.ps1]: No problem.  BTW what you originally had is referred to as `PascalCase` (initial char capitalized) whereas `camelCase` has the initial character lower cased.  I typically use camelCase for local variables and PascalCase for script/globals and function parameters.

Comment (bgelens) [inline src/InvokePlaster.ps1]: Thanks for that info!  That will bring an end to C# developers laughing at me without telling why ;)

**Pull request merged**